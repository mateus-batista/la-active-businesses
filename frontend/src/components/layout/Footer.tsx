/** @jsx jsx */
import { css, jsx } from "@emotion/react";

export function Footer() {
  return (
    <div
      css={css`
        grid-area: footer;
        background-color: #0f2940;
        display: flex;
        align-items: center;
        justify-content: center;
        padding: 0.5rem 1rem;
        font-size: 0.75rem;
        color: white;
      `}
    >
      by Mateus Alfredo Batista for AE Studio
    </div>
  );
}
