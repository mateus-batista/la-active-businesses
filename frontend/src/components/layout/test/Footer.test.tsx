import React from "react";
import { render, screen } from "@testing-library/react";
import { Footer } from "../Footer";

describe("Footer test", () => {
  test("should render correctly", () => {
    expect(render(<Footer />)).toMatchSnapshot();
  });

  test("should contain AE Studio", () => {
    render(<Footer />);

    expect(
      screen.getByText("AE Studio", {
        exact: false,
      })
    ).toBeInTheDocument();
  });
});
