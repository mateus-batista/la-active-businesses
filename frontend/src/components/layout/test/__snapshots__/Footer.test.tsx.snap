// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer test should render correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": .emotion-0 {
  grid-area: footer;
  background-color: #0f2940;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  padding: 0.5rem 1rem;
  font-size: 0.75rem;
  color: white;
}

<body>
    <div>
      <div
        class="emotion-0"
      >
        by Mateus Alfredo Batista for AE Studio
      </div>
    </div>
  </body>,
  "container": .emotion-0 {
  grid-area: footer;
  background-color: #0f2940;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  padding: 0.5rem 1rem;
  font-size: 0.75rem;
  color: white;
}

<div>
    <div
      class="emotion-0"
    >
      by Mateus Alfredo Batista for AE Studio
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
